runfolder/services.py

```diff
@@ -28,11 +28,13 @@ def __setattr__(self, key, value):
 """
 NONE: Not ready for processing or invalid
 READY: Ready for processing
+PENDING: This runfolder has been handed to some other service, and is awaiting having it's status set.
+         It will wait X number of minutes before becoming available again.
 STARTED: Started processing the runfolder
 DONE: Done processing the runfolder
 ERROR: Started processing the runfolder but there was an error
 """
-RunfolderState = Enum(["NONE", "READY", "STARTED", "DONE", "ERROR"])
+RunfolderState = Enum(["NONE", "READY", "PENDING", "STARTED", "DONE", "ERROR"])
```

Comment:
Just a quick question if we want to harmonize with https://github.com/arteria-project/arteria-core/blob/987b7d495ebb756c07e60f9cfc71df4858c9fafc/arteria/web/state.py ?